bot/seasons/christmas/adventofcode.py

```diff
@@ -130,23 +130,36 @@ def __init__(self, bot: commands.Bot):
 
         await ctx.invoke(self.bot.get_command("help"), "adventofcode")
 
-    @adventofcode_group.command(name="notifications", aliases=("notify", "notifs"), brief="Notifications for new days")
-    async def aoc_notifications(self, ctx: commands.Context):
+    @adventofcode_group.command(name="subscribe",
+                                aliases=("sub", "notifications", "notify", "notifs"),
+                                brief="Notifications for new days")
```

Comment:
Isn't our normal multiline invocation style ```py
like(
   this='and',
   that='and',
   this='and that'
)
```
?

Reply:
changed it accordingly